stock_release_channel/models/stock_picking.py

```diff
@@ -111,16 +111,22 @@ def _get_release_channel_possible_candidate_domain_channel(self):
         return [
             ("is_manual_assignment", "=", False),
             ("state", "!=", "asleep"),
+            "|",
+            ("picking_type_ids", "=", False),
+            ("picking_type_ids", "in", self.picking_type_id.ids),
```

Comment:
Is it expected to use picking data in `_get_release_channel_possible_candidate_domain_channel` domain hook? I thought we designed it for basic criteria selection of the channel (like the ones already defined) without relying on current picking. For that we have `_get_release_channel_possible_candidate_domain_picking`.